### PR TITLE
Workload app support + test example that does nothing

### PR DESCRIPTION
**Pull request by KevinFairise2**
### What does this PR do?

Add support for deploying ECS workload apps in the ECS provider.

The feature already exists for Kubernetes provider, we add it for ECS

### CODE REVIEW

File: test/new-e2e/pkg/environments/aws/ecs/ecs.go [hunk 1]
```diff
@@ -164,6 +166,17 @@ func WithoutAgent() ProvisionerOption {
 	}
 }
 
+// WorkloadAppFunc is a function that deploys a workload app to a kube provider
```
Comment (pducolin) [inline test/new-e2e/pkg/environments/aws/ecs/ecs.go:169]: **💬 suggestion**
```suggestion
// WorkloadAppFunc is a function that deploys a workload app to an ECS cluster
```

**Review by pducolin — APPROVED** [review]

**Comment by KevinFairise2** [comment]
> As a follow up, can you also clean the current ECS environment removing AWSEnvironment and other properties that are not needed? Would require updating NPM ECS tests

Sure, discussed offline I'll do that in a follow-up PRs

**Pull request merged**